app/stores/NetworkStatusStore.js

```diff
@@ -277,4 +286,13 @@ export default class NetworkStatusStore extends Store {
     return Date.now() - this._startTime;
   }
 
+  _setupTimeDifferencePolling = () => {
```

Comment:
Not sure about the name for this reaction...

Reply:
eg: `_pollTimeDifferenceWhenConnected`?

Reply:
That was one of my choices. It is also stopping the polling when disconnected.

Reply:
But that is more in line with our naming convention for reactions. Changing it.